Frameworks/UIKit/UIPopoverController.mm

```diff
@@ -14,81 +14,268 @@
+
+- (void)_dismissPopoverAnimated:(BOOL)animated completion:(dispatch_block_t)completion {
+    [_popoverControllerInternal dismissPopoverAnimated:animated completion:completion];
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setDelegate:(id<UIPopoverControllerDelegate>)delegate {
+    [_delegateInternal setDelegate:delegate];
+}
+
+/**
+ @Status Interoperable
+*/
+- (id<UIPopoverControllerDelegate>)delegate {
+    return [_delegateInternal delegate];
 }
 
 /**
  @Status Stub
- @Notes
 */
 - (void)setContentViewController:(UIViewController*)viewController animated:(BOOL)animated {
     UNIMPLEMENTED();
 }
 
 /**
  @Status Stub
- @Notes
+*/
+- (void)setContentViewController:(UIViewController*)viewController {
+    UNIMPLEMENTED();
+}
+
+/**
+ @Status Interoperable
+*/
+- (UIViewController*)contentViewController {
+    return [_popoverControllerInternal contentViewController];
+}
+
+/**
+ @Status Interoperable
+*/
+- (CGSize)popoverContentSize {
+    return [_popoverControllerInternal popoverContentSize];
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setPopoverContentSize:(CGSize)size {
+    [_popoverControllerInternal setPopoverContentSize:size];
+}
+
+/**
+ @Status Interoperable
 */
 - (void)setPopoverContentSize:(CGSize)size animated:(BOOL)animated {
+    [_popoverControllerInternal setPopoverContentSize:size];
+}
+
+/**
+ @Status Interoperable
+*/
+- (BOOL)isPopoverVisible {
+    return [_popoverControllerInternal isPopoverVisible];
+}
+
+/**
+ @Status Interoperable
+*/
+- (NSArray*)passthroughViews {
+    return [_popoverControllerInternal passthroughViews];
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setPassthroughViews:(NSArray*)passthroughViews {
+    return [_popoverControllerInternal setPassthroughViews:passthroughViews];
+}
+
+/**
+ @Status Stub
+*/
+- (UIPopoverArrowDirection)popoverArrowDirection {
     UNIMPLEMENTED();
+    return UIPopoverArrowDirectionAny;
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setPopoverLayoutMargins:(UIEdgeInsets)popoverLayoutMargins {
+    [_popoverControllerInternal setPopoverLayoutMargins:popoverLayoutMargins];
+}
+
+/**
+ @Status Interoperable
+*/
+- (UIEdgeInsets)popoverLayoutMargins {
+    return [_popoverControllerInternal popoverLayoutMargins];
 }
 
 /**
  @Status Stub
 */
-- (UIColor*)backgroundColor {
+- (Class)popoverBackgroundViewClass {
     UNIMPLEMENTED();
-    return [[_backgroundColor copy] autorelease];
+    return _popoverBackgroundViewClass;
 }
 
 /**
  @Status Stub
 */
-- (void)setBackgroundColor:(UIColor*)color {
+- (void)setPopoverBackgroundViewClass:(Class)popoverBackgroundViewClass {
     UNIMPLEMENTED();
-    _backgroundColor.attach([color copy]);
+    [_popoverBackgroundViewClass release];
+    _popoverBackgroundViewClass = [popoverBackgroundViewClass retain];
+}
+
+/**
+ @Status Interoperable
+*/
+- (UIColor*)backgroundColor {
+    return _backgroundColor;
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setBackgroundColor:(UIColor*)color {
+    [_backgroundColor release];
```

Comment:
my original commit to this pull actually undid someone's earlier change making _backgroundColor a StrongId. I've put that code back in the latest commit and also made better use of StrongId in UIPopoverController.mm.